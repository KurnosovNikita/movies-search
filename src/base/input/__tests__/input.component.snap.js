import React from 'react';
import {shallow} from 'enzyme';
import Input from '../input.component';

describe('Input component', () => {
    let props;
    beforeEach(() => {
        props = {
            onInput: jest.fn(),
            onKeyDown: jest.fn(),
            className: 'className',
            containerClassName: 'containerClassName',
            text: 'text',
            id: 'id',
            placeholder: 'placeholder',
        };
    });

    it('should render <Input> correctly', () => {
        const component = shallow(<Input {...props}/>);

        expect(component).toMatchSnapshot();
    });
});
