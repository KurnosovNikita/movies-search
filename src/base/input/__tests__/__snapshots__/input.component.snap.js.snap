// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component should render <Input> correctly 1`] = `
<div
  className="containerClassName"
>
  <input
    className="className"
    id="id"
    onInput={[MockFunction]}
    onKeyDown={[MockFunction]}
    placeholder="placeholder"
  />
</div>
`;
